.circleci/config.yml

```diff
@@ -29,14 +30,14 @@ jobs:
           path: /home/circleci/project/target/inspection-results
 
   validate-with-maven-script:
-    description: "Runs a maven script using the job name as the argument."
+    description: Runs a maven script using the job name as the argument.
     parameters: &script_parameters
       image-name:
         type: string
-        default: "cimg/openjdk:21.0.6"
-        description: "docker image to use"
+        default: cimg/openjdk:21.0.6
+        description: docker image to use
       command:
-        description: "command to run"
```

Comment:
this kind of gives attention to where its really mandatory to use `"double quotes"`.

<img width="619" height="889" alt="Image" src="https://github.com/user-attachments/assets/cb917a4c-1d6a-4dcd-b50f-3623199b0021" />